sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveInspectors.scala

```diff
@@ -1063,6 +1063,9 @@ private[hive] trait HiveInspectors {
       case DateType => dateTypeInfo
       case TimestampType => timestampTypeInfo
       case NullType => voidTypeInfo
+      case dt =>
+        throw new AnalysisException(
+          s"${dt.getClass.getSimpleName.replace("$", "")} cannot be converted to Hive TypeInfo")
```

Comment:
This is related to this reafactoring?

Reply:
> This is related to this reafactoring?

So make this change as a seperated  jira?

Reply:
I think this is a testing issue (2.).

Reply:
> I think this is a testing issue (2.): [#29085 (comment)](https://github.com/apache/spark/pull/29085#issuecomment-662794540)

OK, change in next pr(2)